cms/djangoapps/contentstore/views/program.py

```diff
@@ -27,7 +28,8 @@ def get(self, request, *args, **kwargs):
             return render_to_response('program_authoring.html', {
                 'show_programs_header': programs_config.is_studio_tab_enabled,
                 'authoring_app_config': programs_config.authoring_app_config,
-                'programs_api_url': programs_config.public_api_url,
+                'lms_base_url': '//{}/'.format(settings.LMS_BASE),
+                'programs_api_url': programs_config.public_service_url,
```

Comment:
The authoring app currently expects a base Programs URL. However, the app then appends "/api/v1/" to the URL everywhere it's used; see https://github.com/edx/programs/search?utf8=%E2%9C%93&q=programsBaseUrl&type=Code.

I think it might be cleaner for the app to simply expect a Programs API URL of the form "http://domain:port/api/v1/." Doing so would allow the app to leverage API versioning configuration exposed in Studio, as well as get rid of some duplicated code.